scheduler/controllers/instrumentationconfig_controller.go

```diff
@@ -14,27 +15,26 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
-// InstrumentedApplicationReconciler reconciles a InstrumentedApplication object
-type InstrumentedApplicationReconciler struct {
+type InstrumentationConfigReconciler struct {
 	client.Client
 	Scheme           *runtime.Scheme
 	ImagePullSecrets []string
 	OdigosVersion    string
 }
 
-func (r *InstrumentedApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (r *InstrumentationConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
-	logger.V(0).Info("Reconciling InstrumentedApps")
+	logger.V(0).Info("Reconciling InstrumentationConfig")
 
 	namespace := env.GetCurrentNamespace()
 
-	var instrumentedApps odigosv1.InstrumentedApplicationList
-	err := r.List(ctx, &instrumentedApps)
+	var instrumentedConfigs odigosv1.InstrumentationConfigList
+	err := r.List(ctx, &instrumentedConfigs)
 	if err != nil {
-		logger.Error(err, "failed to list InstrumentedApplications")
+		logger.Error(err, "failed to list InstrumentationConfigs")
 		return ctrl.Result{}, err
 	}
-	numberOfInstrumentedApps := len(instrumentedApps.Items)
+	numberOfInstrumentedApps := len(instrumentedConfigs.Items)
```

Comment:
numberOfInstrumentedApps -> numberOfInstrumentationConfigs

Reply:
It's intentional. Each instrumentation config object represents an app